src/NzbDrone.Core/MediaFiles/MediaFileDeletionService.cs

```diff
@@ -61,8 +61,8 @@ public void DeleteMovieFile(Movie movie, MovieFile movieFile)
 
             if (_diskProvider.GetDirectories(rootFolder).Empty())
             {
-                _logger.Warn("Movie's root folder ({0}) is empty.", rootFolder);
-                throw new NzbDroneClientException(HttpStatusCode.Conflict, "Movie's root folder ({0}) is empty.", rootFolder);
+                _logger.Warn("Movie's root folder ({0}) is empty. Rescan will not update movies as failsafe.", rootFolder);
```

Comment:
```suggestion
                _logger.Warn("Movie's root folder ({0}) is empty. Rescan will not update movies as a failsafe.", rootFolder);
```

Reply:
Ahh missed that, will get it when I get home from the shop